### ISSUE TITLE
Improve TCE menu

### ISSUE THREAD

**Issue by cweiske** [TYPO3-Documentation/TYPO3CMS-Reference-CoreApi#45]
The left navigation menu for the TCE section isn't nice.

I don't know why there is a "Files" section, or why everything is below "Introduction".

![2015-12-18 auswahl_001](https://cloud.githubusercontent.com/assets/59036/11891695/5ab55c46-a561-11e5-9d4b-0c17e10c3470.png)

**Comment by marble** [comment]
Thanks for reporting. I just fixed this one. Wait 5 minutes and reload browser. Problem arises when the .toctree directive (which pulls in the next chapters) is at the bottom of the page and at that point we are noch on the toplevel of headlines of THAT page.

Unfortunately there is no way I know of to reset the include position back to the toplevel. We would need something to tell Sphinx: "Hey, on this page we have sections (=headlines) and subsections, but now assume we are at the toplevel again and pull in the next pages here."

The cure I used here is: Don't have subsections on the page. Instead, I made the "Introduction" part know a file of its own and include it via .toctree - at the same level - like the others.

If you find more of these issues - and I'm sure there are - leave a short note "wrong hierarchy" in the Issue tracker.

Thank you very much for caring!
